src/templates/ScorecardTemplate.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Grid } from '@trussworks/react-uswds';
+
+import Layout from '@/components/Layout';
+import MainGridContainer from '@/components/MainGridContainer';
+import AgencyInfo from '@/components/AgencyInfo';
+import { IAgencyInfo, IDeliverInvest } from '@/types';
+
+interface Props {
+  pageContext: {
+    agencyInfo: IAgencyInfo;
+    deliverInvest: IDeliverInvest;
+  };
+}
+
+const ScorecardTemplate: React.FC<Props> = ({ pageContext }) => {
+  const agencyName = pageContext.agencyInfo.Name;
+
+  return (
+    // location prop is not available during SSR, so removing
```

Comment:
I had to remove the location prop to the Layout component.  This is because this component is provided to the createPages API in gatsby-node and since that done in Node (via SSR), it doesn't have access to the location data. There's some information here:

https://www.gatsbyjs.com/docs/debugging-html-builds/

I think I'm facing #1. It's not clear to me how to work around it yet.

Reply:
Yes exactly! That's what I'm going to do! I saw it on TFH and thought yes this should work!